src/services/actions/Scaffolder.ts

```diff
@@ -147,6 +147,45 @@ export class Scaffolder {
     PnPWebview.postMessage(WebviewCommand.toWebview.validateComponentName, true);
   }
 
+  /**
+	 * Returns the value of the createNodeVersionFileDefaultValue setting and sends it to the webview.
+	 */
+	public static async createNodeVersionFileDefaultValue() {
+		const value = getExtensionSettings<boolean>(
+			'createNodeVersionFileDefaultValue',
+			false
+		);
+
+		PnPWebview.postMessage(
+			WebviewCommand.toWebview.createNodeVersionFileDefaultValue,
+			value
+		);
+	}
+
+	/**
+	 * Returns the value of the nodeVersionManagerFile setting and sends it to the webview.
+	 */
+	public static async nodeVersionManagerFile() {
+		const value = getExtensionSettings<string>('nodeVersionManagerFile', false);
+
+		PnPWebview.postMessage(
+			WebviewCommand.toWebview.createNodeVersionManagerFile,
+			value
+		);
+	}
+
+	/**
+	 * Returns the value of the nodeVersionManager setting and sends it to the webview.
+	 */
+	public static async nodeVersionManager() {
+		const value = getExtensionSettings<string>('nodeVersionManager', false);
```

Comment:
`false` should not be a default value of a setting which we rather expect to return a string. I also introduced a bug fix connected to that just a while ago which will most probably make this not working in order to ensure correct types. 
lets update  this to `nvm` as default
```suggestion
		const value = getExtensionSettings<string>('nodeVersionManager', 'nvm');
```